### DISCUSSION TITLE
Unwanted border and shadow

### DISCUSSION THREAD

**Discussion by kingyiphk** [tailwindlabs/tailwindcss#10594] [Help]
I just start to use next + tailwind to build a online shop.
I just  try to create a simple **<Layout> (with header, main and footer section) component** and use that in index.js.
However, the result show in below. It seems that there is many div with border and shadow on the page. 
Please! Do anybody know what happen there? 
(Remark: I tested tailwind in work and able to change style)

![tailwind](https://user-images.githubusercontent.com/95980752/219124217-767f9ae1-4990-436d-b2dd-fe3eeed65bcb.png)

**Comment by wongjn** [comment]
Seems like you have added a `linear-gradient()` to the `<body>` element. Background images will repeat as seen in your image by default due to the default value of `background-repeat` being `repeat`. You can override this default and stop the repeating gradient by setting `background-repeat: no-repeat` via the `bg-no-repeat` utility class.
